commit 01ec1707a767eaeabec3135a3866ded2920c2ff6
Author: Christoph Moench-Tegeder <cmt@burggraben.net>

    set project type to initialize cmake compiler checks
    
    Qt6 will pull in FindThreads() which fails w/o C compiler

diff --git distutils_cmake/CMakeLists.txt distutils_cmake/CMakeLists.txt
index 976be6a..e59804a 100644
--- distutils_cmake/CMakeLists.txt
+++ distutils_cmake/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.5)
-project(pivy_cmake_setup NONE)
+project(pivy_cmake_setup C CXX)
 
 
 find_package(Coin CONFIG REQUIRED)
